### PR TITLE
Added "confirm it works" sections to all Getting Starteds

### PR DESCRIPTION
**Pull request by heff**

### CODE REVIEW

File: bitmovin/getting-started.md [hunk 1]
```diff
@@ -56,3 +56,12 @@ After creating the instance of the Bitmovin player, simply pass that player refe
   });
 </script>
 ```
+
+## Confirming it works
+
+After you've finished integration, you can quickly see if it's working by opening your [browser developer tools](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools), selecting the network tab, and then clicking 'play' on the video you've set to be monitored. You should begin to see beacons being sent to the mux servers (litix.io) like in the example image.
```
Comment (theRealWardo) [inline bitmovin/getting-started.md]: ...being sent to the mux servers...

I feel like we should capitalize our company name. "Mux" yea?

File: jwplayer/getting-started.md [hunk 2]
```diff
@@ -3,11 +3,11 @@ Title: Getting Started
 Sort: 1
 */
 
-Include the jwplayer-mux file after the core JW Player javascript file in the page. Use the Mux-hosted version of the script to receive automatic updates (the API will not change within major versions). Note that the key in the example should be replaced with the key provided by JW Player for your account.
+Include the jwplayer-mux file after the core JW Player javascript file in the page. Use the Mux-hosted version of the script to receive automatic updates (the API will not change within major versions). Note that the KEY in the example should be replaced with the key provided by JW Player for your account.
```
Comment (theRealWardo) [inline jwplayer/getting-started.md]: maybe make KEY code font? `KEY`

File: bitmovin/getting-started.md (hunk 1 above)
Comment (theRealWardo) [inline bitmovin/getting-started.md]: any reason why we are linking the browser developer tools to a Mozilla site and then including an image from Chrome dev tools? that feels strange to me, but I guess it is ok?

**Review by theRealWardo — APPROVED** [review]
nothing really blocking, but capitalizing our company name would be nice.

File: bitmovin/getting-started.md (hunk 1 above)
Comment (heff) [reply bitmovin/getting-started.md]: Yeah, I can see the confusion, but the mozilla page is the best overview of all tools I can find, not just specific to FF.

**Comment by heff** [comment]
Addressed the comments. Should be good now.

**Pull request merged**